cli/src/app.rs

```diff
@@ -245,6 +245,13 @@ pub fn add_subcommands(command: Command) -> Command {
                                 .long("json")
                                 .help("Produce output in json format (default: false)"),
                         ),
+                )
+                .subcommand(
+                    Command::new("revoke-token").about("Revoke an API token").arg(
+                        Arg::new("token-name")
+                            .action(ArgAction::Append)
+                            .help("Unique token names which identify the tokens"),
```

Comment:
```suggestion
                        Arg::new("token-name")
                            .value_name("TOKEN-NAME")
                            .action(ArgAction::Append)
                            .help("Unique token names which identify the tokens"),
```

Adding `.value_name()` will fix the missing entry in the corresponding markdown doc for this command.